activerecord/lib/active_record/railtie.rb

```diff
@@ -39,8 +39,6 @@ class Railtie < Rails::Railtie # :nodoc:
     rake_tasks do
       namespace :db do
         task :load_config do
-          ActiveRecord::Tasks::DatabaseTasks.database_configuration = Rails.application.config.database_configuration
```

Comment:
This was first added way back in 2013 as part of https://github.com/rails/rails/commit/d1d7c86d0c8dcb7e75a87644b330c4e9e7d6c1c1 but this is no longer loaded because [`ActiveRecord::Base.configurations` is set once `ActiveRecord` is loaded](https://github.com/rails/rails/blob/7da82b0fc32a3257d9bc44605783d8aa1a7474a1/activerecord/lib/active_record/railtie.rb#L219).